app/utils/bundle-importer.js

```diff
@@ -353,7 +353,7 @@ YUI.add('bundle-importer', function(Y) {
           ghostService.set('config', config);
 
           this.env.deploy(
-              record.args[0],
+              charm.get('id'),
```

Comment:
+1 this is the meat and potatoes of this fix so good to comment on why it's strange.